src/modules/form/components/DynamicField.tsx

```diff
@@ -306,8 +313,9 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
             onChange={onChangeEvent}
             horizontal={horizontal}
             currency={item.type === ItemType.Currency}
-            inputWidth={width}
+            inputWidth='120px'
```

Comment:
For `NumberInput`, the `inputWidth` was not getting passed down, and instead the width was getting set [here](https://github.com/greenriver/hmis-frontend/blob/release-108/src/components/elements/input/TextInput.tsx#L52-L55)  in the `TextInput` component. However, that is skipped now that we added the `sx` prop to this component, so I set it manually here. Let me know if I should take a different approach -- I saw that adding ` && !sx` in the linked snippet was a recent change